src/main/java/com/google/api/codegen/config/PageStreamingConfig.java

```diff
@@ -218,4 +221,45 @@ static PageStreamingConfig createPageStreamingFromProtoFile(
     }
     return new AutoValue_PageStreamingConfig(pagingFields, resourcesFieldConfig);
   }
+
+  static PageStreamingConfig createPageStreamingConfig(
```

Comment:
Why package-private?

Reply:
oh this was just under the principle that we should limit visibility everywhere where possible; we can expand scope later if needed

Reply:
also intellij waves a little yellow flag

Reply:
Fair enough :-) I don't know if we have a convention throughout gapic-generator, but one thing to consider is adding a comment indicating why it is package-private (something like "Package-private to allow access from GapicMethodConfig") because otherwise it can look like an omission of visibility rather than a choice. Totally optional though.